types/jest/index.d.ts

```diff
@@ -26,6 +26,7 @@
 //                 Pawel Fajfer <https://github.com/pawfa>
 //                 Alexandre Germain <https://github.com/gerkindev>
 //                 Adam Jones <https://github.com/domdomegg>
+//                 Tom Mrazauskas <https://github.com/mrazauskas>
```

Comment:
It would be nice to be notified if types are changing here, so I could adjust types in Jest repo.